.github/workflows/cicd.yml

```diff
@@ -0,0 +1,44 @@
+name: Create and publish a Docker image
```

Comment:
probably should rename the file from `cicd.yml` to build or something similar.

Reply:
@ryanluker can do, can you suggest why?

Reply:
Oh just that cicd.yml is more around testing and deployment of app stuff.
AFAIK this workflow is purely about building and publishing the docker image so that doesn't totally match?
`build_and_publish.yml` maybe?